pkg/cbfs/fsp.go

```diff
@@ -0,0 +1,33 @@
+package cbfs
+
+import (
+	"io"
+	"log"
+)
+
+func init() {
+	if err := RegisterFileReader(&SegReader{Type: TypeFSP, Name: "fsp", New: NewFSP}); err != nil {
+		log.Fatal(err)
+	}
+}
+
+func NewFSP(f *File) (ReadWriter, error) {
```

Comment:
you'll need to add comments for all this, but golint will doubtless complain for me :-)

Reply:
done